examples/portworx/portworx_with_iam_policy/README.md

```diff
@@ -0,0 +1,198 @@
+# Portworx add-on for EKS Blueprint
```

Comment:
I think we can remove 2 examples and just provide one with IAM policy instead.

Reply:
So keeping IRSA aside, for both the ways we considered providing a sample blueprint as both ways have quite different steps that need to be carried